support-frontend/assets/pages/[countryGroupId]/checkout.tsx

```diff
@@ -657,10 +657,14 @@ function CheckoutComponent({ geoId }: Props) {
 			if (processPaymentResponse.status === 'success') {
 				const order = {
 					firstName: personalData.firstName,
+					email: personalData.email,
 					price: price,
 					product: productId,
 					ratePlan: query.ratePlan,
 					paymentMethod: paymentMethod as string,
+					// TODO - get this from the /identity/get-user-type endpoint (new/guest/existing)
+					userTypeFromIdentityResponse: 'new',
+					csrf: csrf,
```

Comment:
Similarly we can read this on the thank you page from `window.guardian.csrf.token`.

Reply:
Can see its not worth duplicating in session storage, updated.